apps/srt-live-transmit.cpp

```diff
@@ -182,6 +182,7 @@ int main( int argc, char** argv )
         cerr << "\t-r:<report-frequency=0> - bandwidth report frequency\n";
         cerr << "\t-s:<stats-report-freq=0> - frequency of status report\n";
         cerr << "\t-pf:<format> - printformat (json or default)\n";
+        cerr << "\t-statsreport:<filename> - stats report file name (cout for output to cout, or a filename)\n";
```

Comment:
This option is about defining a file for stats output; wouldn't `-sf` and `-statsfile` sound better?

Reply:
Renamed to `statsout:[stdout|filename]`